torch_xla/core/xla_model.py

```diff
@@ -90,12 +90,17 @@ def xrt_world_size(defval=1):
   Returns:
     The number of devices which is taking part of the replication.
   """
-  if pjrt.using_pjrt():
-    return pjrt.world_size()
-
-  return xu.getenv_as(xenv.WORLD_SIZE, int, defval=defval)
+  global g_xrt_world_size
+  if g_xrt_world_size is not None:
+    return g_xrt_world_size
 
+  if pjrt.using_pjrt():
+    g_xrt_world_size = pjrt.world_size()
```

Comment:
@will-cromar to double check, I believe `world_size` will never changed after the first init, but correct me if I am wrong.

Reply:
That's correct. This call will force the PJRT client to initialize, but the devices and world size will not change after that.

Reply:
@will-cromar Do you know if there is a place in pjrt where we can call xrt_world_size and get_ordinal to cache the values?